app/assets/stylesheets/darkswarm/shop.css.scss

```diff
@@ -11,18 +11,111 @@
 @import "shop-taxon-flag";
 @import "shop-popovers";
 
+$sidebar-small-width: 85%;
+$sidebar-medium-width: 65%;
+$sidebar-large-width: 45%;
+$sidebar-footer-height: 5em;
+
 .darkswarm {
+  .shop-filters-sidebar {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+
+    .background {
+      position: fixed;
+      top: 0;
+      right: 0;
+      z-index: 200;
+      height: 100%;
+      width: 100%;
+      background-color: $shop-sidebar-overlay;
+      opacity: 0;
+      transition: opacity $transition-sidebar;
+    }
+
+    &.shown {
+      .background {
+        opacity: 1;
+      }
+
+      .sidebar, .sidebar-footer {
+        margin-right: 0;
+      }
+    }
+
+    .sidebar {
+      position: fixed;
+      top: 0;
+      right: 0;
+      z-index: 210;
+      height: 100%;
+      width: $sidebar-large-width;
+      margin-right: -$sidebar-large-width;
+      background-color: rgba($white, 0.95);
+      padding: 1em;
+      transition: margin $transition-sidebar;
+      overflow-y: scroll;
+
+      .property-selectors {
+        margin-bottom: $sidebar-footer-height + 2em;
+      }
+    }
+
+    .sidebar-footer {
+      background-color: $grey-800;
+      width: $sidebar-large-width;
+      margin-right: -$sidebar-large-width;
+      height: $sidebar-footer-height;
+      position: fixed;
+      bottom: 0;
+      right: 0;
+      transition: margin $transition-sidebar;
+      padding: 1em;
+
+      button {
+        height: 3em;
+        background-color: $grey-800;
+        color: $white;
+        border: 1px solid $grey-600;
+        border-radius: $radius-medium;
+        width: 47%;
+        margin: 0;
+        padding: 0;
+
```

Comment:
yesterday I was styling a close button on the product modal, I wonder if this :point_up: is not a generic button style that can be re-used later.

Reply:
I've extracted these button styles :+1: